backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -128,4 +128,18 @@ SearchException conflictingFieldCodecsForSearch(String absoluteFieldPath,
 	@Message(id = 519, value = "Attempt to unwrap an Elasticsearch backend to %1$s,"
 			+ " but this backend can only be unwrapped to %2$s." )
 	SearchException backendUnwrappingWithUnknownType(Class<?> requestedClass, Class<?> actualClass);
+
+	@Message(id = 520, value = "The index schema node '%2$s' was added twice at path '%1$s'."
+			+ " Multiple bridges may be trying to access the same index field, "
+			+ " or two indexed-embeddeds may have prefixes that lead to conflicting field names,"
+			+ " or you may have declared multiple conflicting mappings."
+			+ " In any case, there is something wrong with your mapping and you should fix it." )
+	SearchException indexSchemaNodeNameConflict(String absolutePath, String name);
+
+	@Message(id = 521, value = "Field name '%1$s' is invalid: field names cannot be null or empty." )
+	SearchException relativeFieldNameCannotBeNullOrEmpty(String relativeFieldName);
+
+	@Message(id = 522, value = "Field name '%1$s' is invalid: field names cannot contain a dot ('.')."
+			+ " If you want a tree of fields, declare object field using the objectField() method." )
```

Comment:
Not sure this error message is helpful. The end user will have it while having a dot in his field name but has no clue about `objectField()`. I think we should just state that they can't use dots.

Reply:
The second part would have been useful to people implementing their own bridge, but as you will. I removed it.

Reply:
Well you're right in the case of field bridges, it might be useful. Maybe we should be more clear with something as:
`Field name '%1$s' is invalid: field names cannot contain a dot ('.'). Either remove the dot from your field name or if you are using a field bridge and want a tree of fields, declare object field using the objectField() method.`

Reply:
Done!